src/deployment/default/components/alert.ts

```diff
@@ -3,6 +3,7 @@ import { StyleConfig, StyleFunctionProps } from "@chakra-ui/theme-tools";
 
 const Alert: StyleConfig = {
   variants: {
+    // TODO: reconsider this theme customisation
```

Comment:
Is there something to reconsider?

Reply:
It seems odd to pull in teal but green on green isn't great so I've left it pending better ideas.